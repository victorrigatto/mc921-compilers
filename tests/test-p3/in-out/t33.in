int n = 3;

int doubleMe (int x) {
    return x * x;
}

void main ()
{
    int v = n;
    v = doubleMe(v, 2);
    assert v + n * n;
    return;
}
