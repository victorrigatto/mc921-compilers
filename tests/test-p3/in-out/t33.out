SemanticError: no. arguments to call doubleMe function mismatch @ 10:9
